tests/foreman/virtwho/api/test_nutanix.py

```diff
@@ -0,0 +1,188 @@
+"""Test class for Virtwho Configure API
+
+:Requirement: Virt-whoConfigurePlugin
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Virt-whoConfigurePlugin
+
+:Assignee: yanpliu
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+from nailgun import entities
+
+from robottelo.cli.host import Host
+from robottelo.cli.subscription import Subscription
+from robottelo.config import settings
+from robottelo.virtwho_utils import deploy_configure_by_command
+from robottelo.virtwho_utils import deploy_configure_by_script
+from robottelo.virtwho_utils import get_configure_command
+from robottelo.virtwho_utils import get_configure_file
+from robottelo.virtwho_utils import get_configure_option
+
+
+@pytest.fixture()
+def form_data(default_org, target_sat):
```

Comment:
I'd agree its really not necessary to use a fixture here - but won't be blocking the PR on it. It's something that was done during pytest conversion of the framework, but isn't a great pattern to continue carrying forward. 

Making the `settings` references several times can get annoying, but this can certainly just be a normal function, not a fixture.  There's no setup happening.

Reply:
> I'd agree its really not necessary to use a fixture here - but won't be blocking the PR on it. It's something that was done during pytest conversion of the framework, but isn't a great pattern to continue carrying forward.
> 
> Making the `settings` references several times can get annoying, but this can certainly just be a normal function, not a fixture. There's no setup happening.

@mshriver Thanks for your suggestion , now all the virt-who plugin cases have this format, in the future, we will research update this patttern.